pkg/ocm/client.go

```diff
@@ -99,14 +104,16 @@ func (c *Client) NewConnection() (*sdkClient.Connection, error) {
 	} else if c.config.SelfToken != "" {
 		builder = builder.Tokens(c.config.SelfToken)
 	} else {
-		return nil, fmt.Errorf("Can't build OCM client connection. No Client/Secret or Token has been provided")
+		return fmt.Errorf("Can't build OCM client connection. No Client/Secret or Token has been provided.")
```

Comment:
please use the errors package 
```suggestion
		return errors.Errorf("Can't build OCM client connection. No Client/Secret or Token has been provided.")
```
it have a stuck trace option.

Reply:
Done